src/components/NavigationBar.tsx

```diff
@@ -27,6 +31,7 @@ const NavigationBar: React.FC<NavigationBarProps> = (props) => {
   const [showSettingsModal, setShowSettingsModal] = useState<boolean>(false);
   const [songbookName, setSongbookName] = useState<string>(defaultNavigationTitle);
   const { bookId } = useParams<{ bookId: string }>();
+  const [songPageBlobUrl, setSongPageBlobUrl] = useState<string>("");
```

Comment:
remove if possible

Reply:
the empty string `""`

Reply:
it will default to undefined, don't think we want that